api-docs/swagger-v1.1.yml

```diff
@@ -1322,6 +1323,13 @@ parameters:
     in: query
     description: restrict by usage search query (name)
     required: false
+  usages_flat:
+    name: flat
+    type: boolean
+    in: query
+    description: format the returned set of usages - which by default is arranged in a tree structure - as a flat list; pagination options only apply if this option is true
+    default: false
+    required: false
```

Comment:
Zullen we dit een interne optie maken of niet? Hij voelt niet helemaal of (omdat nesting level niet aanwezig is).